### PR TITLE
Add blog post about the 1.9.0 release

### PR DESCRIPTION
**Pull request by flavio**
Kubewarden 1.9.0 blog post

### CODE REVIEW

File: content/blog/2023/11/kubewarden-1.9-release.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+receive.
+
+Initially, this feature was available only to policies written using traditional programming languages. However, starting
+from the 1.9.0 release, the policies written using Rego can use cluster information.
+
+Kubewarden can run policies written in Rego that target both the
+[Gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/) and the
+[`kube-mgmt`](https://github.com/open-policy-agent/kube-mgmt)
+platforms. In both cases, the Kubernetes information is shared with the policy using the same format used by these
+platforms. That means there's no need to rewrite your Gatekeeper/kube-mgmt policies. 😎
```
Comment (viccuad) [inline content/blog/2023/11/kubewarden-1.9-release.md:26]: :D.

This reminds me we can update the Gatekeeper comparison.

Comment (flavio) [reply content/blog/2023/11/kubewarden-1.9-release.md:26]: You're right! I'm on it

File: content/blog/2023/11/kubewarden-1.9-release.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+## Validate generic requests
+
+Initially, Kubewarden was created to validate Kubernetes admission requests. However, our community has recently
+expressed an interest in reusing the Kubewarden policy platform to write validation/mutation policies not specific to
+Kubernetes.
+
+Starting from the 1.9.0 release, we introduce the concept of _"raw policies"_. These are regular Kubewarden policies that
+validate arbitrary JSON objects. They could validate Terraform plans, GitHub Actions, AWS CloudFormation resources &hellip;
+
+More details about raw policies can be found [here](https://docs.kubewarden.io/howtos/raw-policies).
```
Comment (viccuad) [inline content/blog/2023/11/kubewarden-1.9-release.md:46]: I find the generic request paragraph somewhat short in info. Maybe adding a line explaining that this was achieved by providing the policy-server with a new `/validate_raw` endpoint, and that one can make use of the policy-server on its own to run this new raw policies.

**Review by viccuad — APPROVED** [review]
LGTM! just a minor question for more info.

File: content/blog/2023/11/kubewarden-1.9-release.md (hunk 1 above)
Comment (flavio) [reply content/blog/2023/11/kubewarden-1.9-release.md:26]: I forgot I've already done it :)

https://docs.kubewarden.io/next/explanations/opa-comparison

File: content/blog/2023/11/kubewarden-1.9-release.md (hunk 2 above)
Comment (flavio) [reply content/blog/2023/11/kubewarden-1.9-release.md:52]: I've expanded the section

File: content/blog/2023/11/kubewarden-1.9-release.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Validate generic requests
+
+Initially, Kubewarden was created to validate Kubernetes admission requests. However, our community has recently
+expressed an interest in reusing the Kubewarden policy platform to write validation/mutation policies not specific to
+Kubernetes.
+
+Starting from the 1.9.0 release, we introduce the concept of _"raw policies"_. These are regular Kubewarden policies that
+validate arbitrary JSON objects. They could validate Terraform plans, GitHub Actions, AWS CloudFormation resources &hellip;
+The are exposed by the Kubewarden Policy Server using a the dedicated `/validate_raw` endpoint.
```
Comment (jhkrug) [inline content/blog/2023/11/kubewarden-1.9-release.md:45]:
```suggestion
They are exposed by the Kubewarden Policy Server using the dedicated `/validate_raw` endpoint.
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
Some minor suggestions.

**Comment by flavio** [comment]
Thanks @jhkrug, I've applied all your suggestions

**Review by jvanz — APPROVED** [review]

**Pull request merged**